core/domain/collection_domain_test.py

```diff
@@ -703,33 +731,35 @@ def test_yaml_import_and_export(self):
         self.assertEqual(yaml_content_2, yaml_content)
 
         # Should not be able to create a collection from no YAML content.
-        with self.assertRaisesRegex(
+        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
             utils.InvalidInputException,
             'Please ensure that you are uploading a YAML text file, '
             'not a zip file. The YAML parser returned the following error: '
         ):
             collection_domain.Collection.from_yaml('collection3', '')
 
-    def test_from_yaml_with_no_schema_version_specified_raises_error(self):
+    def test_from_yaml_with_no_schema_version_specified_raises_error(
+        self
+    ) -> None:
         collection = collection_domain.Collection(
             self.COLLECTION_ID, 'title', 'category', 'objective', 'en', [],
-            None, [], 0)
+            None, [], 0) # type: ignore[arg-type]
```

Comment:
Why is this ignore needed?

Reply:
Here schema_version is expected to be int but we're providin none value. that's why i added this. and actually this test is all about wrong assigment!!!